docs/en/week08/practicum08.sbv

```diff
@@ -206,7 +206,7 @@ the code is actually made of two things
 it's made of one thing that is this kind
 
 0:03:11.230,0:03:18.819
-of capital e of Z and V of Z and they're
+of capital E of z and V of z and they're
```

Comment:
0:03:02.200,0:03:08.439
having the hidden layer age now we have>>hidden layer h